cli/src/config/misc.toml

```diff
@@ -17,6 +17,7 @@ private-commits = "none()"
 push-bookmark-prefix = "push-"
 push-new-bookmarks = false
 sign-on-push = false
+change-id = false
```

Comment:
nit: this is redundant (because the default is set by `lib/src/config/misc.toml`)